mypyc/ops.py

```diff
@@ -1126,18 +1126,70 @@ def __str__(self) -> str:
         return '\n'.join(format_func(self))
 
 
+# Some notes on the vtable layout:
```

Comment:
Thanks for adding such detailed documentation! The details of the design are non-obvious and it's good that they are collected here in one place.